blocks/testing/include/gnuradio-4.0/testing/NullSources.hpp

```diff
@@ -35,7 +35,7 @@ Commonly used for testing and simulations where consistent output and finite exe
     Annotated<gr::Size_t, "max samples", Doc<"count>n_samples_max -> signal DONE (0: infinite)">> n_samples_max = 0U;
     Annotated<gr::Size_t, "count", Doc<"sample count (diagnostics only)">>                        count         = 0U;
 
-    GR_MAKE_REFLECTABLE(ConstantSource, out, n_samples_max, count);
+    GR_MAKE_REFLECTABLE(ConstantSource, out, default_value, n_samples_max, count);
```

Comment:
Nice catch :+1:

Reply:
Had to add a small followup for this: since default_value was of type T and the Sinks are instantiated with a T=UncertainValue, which is not supported as a setting type i had to change the type to value_t -> cannot set default values with errors or imaginary parts for now.

Reply:
qa_FileIO is working again, but for non-trivial types the sources still cause some conversion errors and unexpected results.